src/tests/stdlib/strcat.c

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file
+ *
+ * @date Mar 31, 2020
+ * @author: Prajawal Kumar Pandey
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("test suite for strcat()");
+
+TEST_CASE("concatenate full string with strcat()") {
+    char src[] = "string";
+    char dst[] = "char ";
+
+    strcat(dst, src);
+    test_assert_not_null(dst);
+    test_assert_zero(strcmp(dst, "char string"));
+}
+
+TEST_CASE("concatenate empty string with strcat()") {
+    char src[] = "";
+    char dst[] = "char ";
+    char chk[sizeof(dst)+1];
+
+    strcpy(chk,dst);
```

Comment:
it's better to use srtncpy() and add space after comma please

Reply:
Done!